includes/admin/class-wcmp-export-consignments.php

```diff
@@ -142,10 +142,13 @@ public function setCustomItems(): void
 
                 // Weight (total item weight in grams)
                 $weight       = (int) round(WCMP_Export::getItemWeight_kg($item, $this->order) * 1000);
+                $parcelWeight = $this->getSetting(WCMP_Settings::SETTING_EMPTY_PARCEL_WEIGHT);
+                $totalWeight  = $parcelWeight + $weight;
+
```

Comment:
Ik zou dit in een functie gooien, iets als dit:

```php
function getTotalWeight(int $weight): int 
{
    $parcelWeight = $this->getSetting(WCMP_Settings::SETTING_EMPTY_PARCEL_WEIGHT);
    return $parcelWeight + $weight;
}
```

Dan heb je geen dubbele code meer in `getTotalPackageWeight`

Reply:
done